sensitive-banner-static/sensitive-banner-js-config.template.php

```diff
@@ -42,6 +42,25 @@
 			form: {
 				formId: 'donationForm',
 				formAction: 'https://test.wikimedia.de/spenden/index.php?piwik_campaign={{{CampaignName}}}&piwik_kwd={{{BannerName}}}'
+			},
+			tracking: {
+				keyword: '{{{BannerName}}}',
+				libUrl: 'https://test.wikimedia.de/piwik.js',
```

Comment:
As I get it, this will be loaded everytime the banner loads. Shouldn't we put that into the stuff we deploy on commons?